shipyard/README.md

```diff
@@ -0,0 +1,52 @@
+# Shipyard
+
+## Overview
+
+The Datadog Shipyard integration passes your [Shipyard][1] ephemeral environment logs to your Datadog dashboard. This is useful for using Datadog's [log analysis and correlation][8] functions and helping your team with [shift-left testing][9].
+
```

Comment:
The overview section should include:
1. What Shipyard does in 1-2 sentences
2. The value that customers would get from this integration 
3. Data you are sending in

Reply:
I believe I have included the necessary details for you here: 0416cd420dbea24e7aebb1351f6ad906b3a7bd41